dictionaries/en_US/src/hunspell/en_US-large.dic

```diff
@@ -19964,6 +19964,7 @@ architectonics/M
 architectural/Y
 architecture/MS
 architrave/SM
+archivable
```

Comment:
@claudiu-cristea,

Thank you for suggesting the addition, but this change will get lost the next time the dictionary is updated from the source.

A better place is: https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_US/src/additional_words.txt

Reply:
or: https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt